.travis.yml

```diff
@@ -45,18 +54,21 @@ before_install: |
     pyenv install $PYTHON 
     pyenv virtualenv $PYTHON MYVERSION
     source ~/.pyenv/versions/MYVERSION/bin/activate
+    pyenv global $PYTHON
```

Comment:
What is pyenv for? If you need to run tests under different Python versions you could use tox and then list different Python versions in the text matrix.

Reply:
This was the only way I knew I could get multiple versions of Python running on Travis for OSX. Will check tox.